src/html/main.html

```diff
@@ -66,6 +66,23 @@
           <div data-braintree-id="apple-pay-button" class="braintree-sheet__button--apple-pay apple-pay-button"></div>
         </div>
       </div>
+      <div data-braintree-id="venmo" class="braintree-venmo braintree-sheet">
+        <div data-braintree-id="venmo-sheet-header" class="braintree-sheet__header">
+          <div class="braintree-sheet__header-label">
+            <div class="braintree-sheet__logo--header">
+              <svg height="24" width="40">
```

Comment:
class = "braintree-icon--bordered" on this svg tag will give it the rounded edges

Reply:
:+1: